app/torii-providers/stripe.js

```diff
@@ -0,0 +1,29 @@
+import Oauth2 from 'torii/providers/oauth2-code';
+import { configurable } from 'torii/configuration';
+import { inject as service } from '@ember/service';
+
+export default Oauth2.extend({
+
+  store: service('store'),
+
+  name    : 'stripe-connect',
+  baseUrl : 'https://connect.stripe.com/oauth/authorize',
+
+  requiredUrlParams : [],
+  optionalUrlParams : ['stripe_landing', 'always_prompt'],
+
+  responseParams: ['code', 'state'],
+
+  clientId: configurable('clientId', async function() {
```

Comment:
@niranjan94 I tried writing my custom provider. Everything seems to work fine. I am able to load clientId from settings here. But when I see the clientId in url (that gets formed), it is `object [Object]` . I printed various outputs of clientId just before return statement and it was proper string with correct clientId. But when url gets build I get it as object [Object]. I tried it for several hours with different options but none worked. I think I am missing something here. Please help.

Reply:
Hey @pradeepgangwar . Let me pull this in locally and try it out.